lib/rules/no-useless-catch.js

```diff
@@ -43,6 +45,7 @@ module.exports = {
 						context.report({
 							node,
 							messageId: "unnecessaryCatchClause",
+							fix: fixer => fixer.remove(node),
 						});
 					} else {
```

Comment:
[Bug] Is there a reason not to also add a `fix` to the `unnecessaryCatch` report? Cases like the following truly are unnecessary, no?

```js
try {
    foo();
} catch (err) {
    throw err;
}
```

To be fixed to:

```js
    foo();
```